mmv1/products/clouddeploy/Target.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Target'
+description: |
+  A Target defines a location to which a Skaffold configuration can be deployed.
```

Comment:
Nit: the description for this resource in our docs reads "The Cloud Deploy `Target` resource". Only reason I mention is because the description you have here slightly confused me.

Reply:
I copied it from [API reference](https://cloud.google.com/deploy/docs/api/reference/rest/v1/projects.locations.targets#Target).

But I would like to avoid confusion, so I updated the description to match the current doc.